### ISSUE TITLE
Implement guard blocked cursor

### ISSUE THREAD

**Issue by Dzierzan** [OpenE2140/OpenE2140#142]
I believe OpenRA engine doesn't support blocked cursor state for guarding. It might require engine update.

@michaeldgg2 yours!

**Comment by michaeldgg2** [comment]
Yes, the blocked cursor is currently hardcoded to `move-blocked`:

https://github.com/OpenRA/OpenRA/blob/3f19c813920363f9bb45b344b313d8819c2cb52f/OpenRA.Mods.Common/Orders/GuardOrderGenerator.cs#L63-L73
